website/src/app/blog/2025-11-28-incident-post-mortem/readme.mdx

```diff
@@ -0,0 +1,56 @@
+In the early morning hours of November 28, 2025 (UTC), we experienced a limited PII exposure affecting a small number of users. The issue **has been resolved**, and this post-mortem explains what happened, what data was involved, how we responded, and what we’re changing to make sure this can’t happen again.
+
+We want to be upfront here: this was **not a product security breach or unauthorized access to our systems**. It was an email routing / mailing-list configuration mistake on our side — but even so, we understand that any exposure of personal information is serious, and we’re sorry this happened.
+
+## What happened
+
+Between **18:30 on November 27, 2025** and **06:15 on November 28, 2025 (UTC)**, an internal mailing-list misconfiguration caused three automated account deletion request emails to be delivered to administrators of other Firezone accounts.
+
+Those emails contained limited PII and account metadata: **user names, email addresses, account names, account slugs, and account IDs.**
+
+## Impact
+
+A total of **three (3) accounts** were affected. The information was visible only to unintended **account administrators** who received the emails during the brief window of misconfiguration — it was not publicly exposed.
+
+We have:
```

Comment:
Do we need to add something like:

"deleted the accounts which requested deletion."

Just to show that any account metadata is now irrelevant.